Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/formatter/TracingStatusHelper.kt

```diff
@@ -13,25 +13,8 @@ object TracingStatusHelper {
 }
 
 /**
- * The following table explains the different stati which can appear in the ui.
- * This follows this prioritization: Tracing, Bluetooth, Connection, Location
- * Connection will only be relevant in one exact case, Bluetooth is relevant in two different cases,
- * but independently from the Connection status. And in every other case Tracing will be shown
- *  except when location is disabled in which case a location related warning will always be shown as it is required for bluetooth tracing.
- *
- * | Tracing | Bluetooth | Connection | Result     |
- * |---------|-----------|------------|------------|
- * | OFF     | OFF       | ON         | TRACING*   |
- * | OFF     | ON        | OFF        | TRACING*   |
- * | OFF     | ON        | ON         | TRACING*   |
- * | OFF     | OFF       | OFF        | TRACING*   |
- * | ON      | ON        | ON         | TRACING    |
- * | ON      | OFF       | ON         | BLUETOOTH  |
- * | ON      | OFF       | OFF        | BLUETOOTH  |
- * | ON      | ON        | OFF        | CONNECTION |
- * *circle has to be disabled via another formatter
- *
- * NB --- If location is disabled, the status will always show a location related warning, regardless of the other connectivity states.
+ * The following table explains the different static which can appear in the ui.
```

Comment:
This PR removes the very table referenced by this comment.
```suggestion
 * The following code decides in which state the ui has to be in.
```